Docs: Updated Vyper guide to use components and added troubleshooting section. (#268)
rsksmart/devportal: rif/rns/libs/javascript/Errors.md

**Message**: The owner of the TLD does not implement the available method

**Description**: The top level domain owner should be a contract that implements the [available](/rif/rns/architecture/rsk-registrar/rskowner#available) method.

## `KB022`

**Message**: No reverse registrar

> **Description**: The owner of `addr.reverse` node has not been set. More information about reverse operations [here](/rif/rns/architecture/ReverseRegistrar).

Reviewer: ```suggestion
**Description**: The owner of `addr.reverse` node has not been set. More information about reverse operations in [Reverse Registar](/rif/rns/architecture/ReverseRegistrar).
```

## `KB023`

**Message**: The reverse registrar does not implement setName method

> **Description**: The owner of `addr.reverse` (Reverse Registrar) should implement [setName](/rif/rns/architecture/ReverseRegistrar#setname) method.

Reviewer: ```suggestion
**Description**: The owner of `addr.reverse` (Reverse Registrar) should implement [`setName`](/rif/rns/architecture/ReverseRegistrar#setname) method.
```

## `KB024`

**Message**: The resolver does not implement setChainAddr method

> **Description**: Domain resolver should implement [setChainAddr](/rif/rns/architecture/MultiCryptoResolver#setchainaddr) interface.

Reviewer: ```suggestion
**Description**: Domain resolver should implement [`setChainAddr`](/rif/rns/architecture/MultiCryptoResolver#setchainaddr) interface.
```
